content/source/docs/cloud/registry/using.html.md

```diff
@@ -55,6 +55,16 @@ Click a module's "Details" button to view its details page. Use the "Versions" d
 
 ![Terraform Cloud screenshot: a module details page](./images/publish-module-details.png)
 
+### Viewing Nested Modules
```

Comment:
Let's put a blank line between the header and the paragraph. Markdown processors and editors handle things more consistently if you make sure there's space between block-level elements. 

```suggestion
### Viewing Nested Modules

```

Reply:
I have added the blank line.